source/includes/APIReference/CustomFields/Examples/_create-request.md.erb

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Indent 2 spaces (as in the update example) for better use of limited real estate.

Reply:
Ok, I think the indentation matches now 👍